gobblin-yarn/src/main/java/org/apache/gobblin/yarn/YarnAutoScalingManager.java

```diff
@@ -128,6 +138,11 @@ public YarnAutoScalingManager(GobblinApplicationMaster appMaster) {
 
     this.autoScalingExecutor = Executors.newSingleThreadScheduledExecutor(
         ExecutorsUtils.newThreadFactory(Optional.of(log), Optional.of("AutoScalingExecutor")));
+
+    this.helixInstanceTags = ConfigUtils.getString(config,
```

Comment:
I guess these are common tags for all Helix instances. If so, can we add a comment here for clarity?